addons/web/static/tests/views/search_panel_tests.js

```diff
@@ -630,7 +630,7 @@ QUnit.module('Views', {
                 id: 777,
                 active: true,
             }
-        ];
+        );
```

Comment:
why not keeping this as before?

Reply:
I need to double check, it might be a leaked change intended only for mail data, diff might be useless on this one.

The reasoning for "push" in data is that it is equivalent of doing "create" in python, which is a natural way to create data in tests. Overriding all data records is the equivalent of clearing the whole target table before doing a create, which should never be done in Python, especially for some models such as company or partners, were initial data are necessary for the system to work.

Reply:
We are not in Python. We are defining a test case, and we setup appropriate data for this test. If we want to use the common data, we can. If we want to force specific data, and do not use the common one, we can as well. In this test, we choose to do the latter, for whatever reason. This is totally fine, imho.